lms/djangoapps/grades/management/commands/send_segment_events_for_failed_learners.py

```diff
@@ -0,0 +1,120 @@
+"""
+Send segment events for failed learners.
+"""
+
+
+import logging
+from datetime import timedelta
+
+from django.core.management.base import BaseCommand
+from django.core.paginator import Paginator
+from django.utils import timezone
+
+from common.djangoapps.course_modes.models import CourseMode
+from common.djangoapps.student.models import CourseEnrollment
+from common.djangoapps.track import segment
+from lms.djangoapps.grades.models import PersistentCourseGrade
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+log = logging.getLogger(__name__)
+
+PAID_ENROLLMENT_MODES = [
+    CourseMode.MASTERS,
+    CourseMode.VERIFIED,
+    CourseMode.CREDIT_MODE,
+    CourseMode.PROFESSIONAL,
+    CourseMode.NO_ID_PROFESSIONAL_MODE,
```

Comment:
How do we know these are the paid mode? Do we have this list defined somewhere before?

Reply:
I think we have already defined this list here: https://github.com/openedx/edx-platform/blob/ceef9bfb1e0df24bf43762c15a0848bf17b8dc9d/lms/djangoapps/grades/events.py#L221

Can we move this list to a common place so that we do not have to repeat it?